### ISSUE TITLE
Notes and clubpenalty

### ISSUE THREAD

**Issue by baem2** [maieul/ledmac#317]
This is an issue very similar to #316, but as it is different, I'm posting it separately: When I have clubpenalty set top 10000 (widowpenalty unchanged) and the last line empty in the first paragraph, notes seem to be completely ignored on that page (not on the other page).

Here's the MWE:
```
\listfiles
\documentclass{book}
\usepackage[german]{babel} 
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{blindtext}
\usepackage[series={},noeledsec,noledgroup]{eledmac}
\usepackage{eledpar}

\clubpenalty = 10000

\begin{document}

\numberpstarttrue

\begin{pages}
\begin{Leftside}
\beginnumbering
\pstart
Dies hier ist ein Blindtext zum Testen von Textausgaben.\ledleftnote{Anm.} Wer diesen
Text liest, ist selbst schuld. Der Text gibt lediglich den Grauwert der Schrift an.
Ist das wirklich so? Ist es gleichgültig, ob ich schreibe: „Dies ist ein Blindtext“
oder „Huardest gefburn“? Kjift – mitnichten! Ein Blindtext bietet mir wichtige
Informationen. An ihm messe ich die Lesbarkeit\ledleftnote{Und Anm.} einer Schrift, ihre Anmutung,
wie harmonisch die Figuren zueinander stehen und prüfe, wie breit oder schmal
sie läuft. Ein Blindtext sollte möglichst viele verschiedene Buchstaben enthalten
und in der Originalsprache gesetzt sein. Er muß keinen Sinn ergeben, sollte
aber lesbar sein. Fremdsprachige Texte wie „Lorem ipsum“ dienen nicht dem
eigentlichen Zweck, da sie eine falsche Anmutung vermitteln.
%the following line is empty – or not ...

\pend

\endnumbering   
\end{Leftside}

\begin{Rightside}
\beginnumbering
\pstart
Dies hier ist ein Blindtext zum Testen von Textausgaben.\ledleftnote{Anm.} Wer diesen
Text liest, ist selbst schuld. Der Text gibt lediglich den Grauwert der Schrift an.
Ist das wirklich so? Ist es gleichgültig, ob ich schreibe: „Dies ist ein Blindtext“
oder „Huardest gefburn“? Kjift – mitnichten! Ein Blindtext bietet mir wichtige
Informationen. An ihm messe ich die Lesbarkeit\ledleftnote{Und Anm.} einer Schrift, ihre Anmutung,
wie harmonisch die Figuren zueinander stehen und prüfe, wie breit oder schmal
sie läuft. Ein Blindtext sollte möglichst viele verschiedene Buchstaben enthalten
und in der Originalsprache gesetzt sein. Er muß keinen Sinn ergeben, sollte
aber lesbar sein. Fremdsprachige Texte wie „Lorem ipsum“ dienen nicht dem
eigentlichen Zweck, da sie eine falsche Anmutung vermitteln.
\pend

\endnumbering
\end{Rightside}
\end{pages}
\Pages
\end{document}
```

**Comment by maieul** [comment]
please try branch issue317 which should solve both #316 and #317

**Comment by baem2** [comment]
This is great! It fixes issue #316 and #317 as well as the problems in my larger document! Thank you!
